specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/examples/OperationSchema_Register.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "endpoint": "mynamespace.servicebus.windows.net",
+    "groupName": "sample-sr-group",
+    "schemaName": "sample-schema-name",
+    "schemaContent": "{\"type\":\"record\",\"name\":\"ExampleUser\",\"namespace\":\"com.azure.schemaregistry.samples\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favoriteNumber\",\"type\":\"int\"}]}",
+    "api-version": "2021-10"
+  },
+  "responses": {
+    "204": {
+      "headers": {
```

Comment:
should the responses here and `OperationSchema_QueryIdByContent` also include content-type?

Reply:
A `204` response means "No Content", so in this specific case there should be no `content-type` header since there is no content.

But even for responses that contain content, there is no need to document the `content-type` header -- it is a standard HTTP header so it can be assumed to be sent when appropriate.